Makefile

```diff
@@ -33,7 +33,7 @@ goodbye:
 #-------#
 # Local #
 #-------#
-dev: hello init_backend up import_sample_data refresh_product_tags
+dev: hello build init_backend _up import_sample_data refresh_product_tags
```

Comment:
Is _up a typo?

Reply:
ah no, I see it's defined after. Why the "_" before?

Reply:
Because up is "build + _up"
While dev is "build + init_backend + _up + ..."

leading _ in python means "internal"